Modules/Devices/test/Devices.API.Tests.Integration/StepDefinitions/GetIdentityListStepDefinitions.cs

```diff
@@ -26,13 +27,9 @@ public async Task WhenAGETRequestIsSentToTheIdentitiesEndpointAsync()
             _identitiesResponse.Should().NotBeNull();
             _identitiesResponse!.Data.Should().NotBeNull();
             _identitiesList = _identitiesResponse!.Data!.Result;
-        }
 
-        [Then(@"the response status code is (\d\d\d) \((?:[a-z]|[A-Z]|\s)+\)")]
-        public void ThenTheResponseStatusCodeIsOK(int code)
-        {
-            _identitiesList.Should().NotBeNull();
-            ((int)_identitiesResponse!.StatusCode).Should().Be(code);
+            _scenarioContext["Data"] = _identitiesList;
+            _scenarioContext["ResponseStatus"] = _identitiesResponse!.StatusCode;
```

Comment:
This isn't type safe. It's easy to have a typo in the key name, which potentially leads to loads of errors where we don't know where they come from. You should use a custom context instead. Stick to the link I sent you last Friday (https://docs.specflow.org/projects/specflow/en/latest/Bindings/Context-Injection.html)